README.md

```diff
@@ -2,33 +2,46 @@
 
 This repository is the central repository for the Chef server.
 
-Currently this repository does not contain any code, as the Chef server
-code is spread out across several repositories.
+If you need to file an issue against another Chef project, you can
+find a list of projects and where to file issues in the
+[community contributions section](https://docs.chef.io/community_contributions.html#issues-and-bug-reports)
+of the [Chef docs](https://docs.chef.io).
```

Comment:
Should say something like, "If you want to file an issue about Chef Server, you're in the right place."